mockacino supports status codes via config file

grzegorzleszek opened the pull request:
let me know what u think ;)

grzegorzleszek committed:
mockacino supports status codes via config file

michaelarmstrong commented on mockacino.rb:
```diff
@@ -95,6 +100,12 @@ def self.listen_address
+  filePath = File.join(MockacinoConfig.site_root, MockacinoConfig.post_path, path, extension, MockacinoConfig.config_file)
+  if File.exist?(filePath)
+      file = File.read(filePath)
+      status file
```
looks good, i guess only 2 things i'd change, rename the default config to config.conf or config.txt just so that people's text editors automatically will support it.

Also, right now it only supports hard coding a status code in there right? it'd be better if it was key value pair based file so its extensible in the future.

So instead of the contents being just '404' for example, the contents could be statusCode = '404' or similar... That way we can expand it in the future without breaking old features.

What do you think ?

grzegorzleszek replied:
Sounds great.
Now it is key value based, so example config.conf will be :
status, 204

grzegorzleszek committed:
config.config + key value pair based

Config file name is "config.conf";
config file is key value pair based, eg:

status, 204
